src/codeu/chat/common/Hub.java

```diff
@@ -1,147 +0,0 @@
-// Copyright 2017 Google Inc.
```

Comment:
Oh my, you are right. Thanks for the catch. It was building and running for me, but I must have never cleaned my build. There could have been an old copy of Hub.class that allowed it to run. I will update the relay to use the timeline as well.